packages/orchestrator/internal/sandbox/fc/process.go

```diff
@@ -154,12 +164,12 @@ func (p *Process) Start(
 		for scanner.Scan() {
 			line := scanner.Text()
 
-			sbxlogger.I(p).Info("stdout: "+line, zap.String("sandbox_id", p.metadata.SandboxId))
+			p.logger.Info("stdout: "+line, zap.String("sandbox_id", p.metadata.SandboxId))
```

Comment:
called this just logger b/c there is only the internal logger here but it might be confusing